site/tests/app/libraries/PollUtilsTester.php

```diff
@@ -47,12 +51,13 @@ public function testExportDataWithNonEmptyPolls() {
                 "Poll #3",
                 "Is this the fourth poll?",
                 "multiple-response",
+                new DateInterval("PT1M30S"),
                 new DateTime("2020-01-13"),
                 "when_ended",
                 "when_ended"
             ),
         ];
-
+        $polls[0]->setClosed();
```

Comment:
Same here.  Why is this change necessary?

Reply:
Same reason as above poll endDate is initialized at beginning of Time.

Reply:
nevermind